logging/config/cluster-logging-tolerations.adoc

```diff
@@ -1,31 +1,23 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: cluster-logging-tolerations
-[id="cluster-logging-tolerations"]
-= Using tolerations to control OpenShift Logging pod placement
 include::_attributes/common-attributes.adoc[]
+[id="cluster-logging-tolerations"]
+= Using tolerations to control logging pod placement
```

Comment:
The principles of  tolerations is a common kubernetes concept.  Instead of the explainations below is there value in referencing the general info about this principle?

Reply:
Probably, but I was trying to just update / unpick what we have, as opposed to overhauling the whole section about this, since the Jira issue was specifically about the collector config for this.

Reply:
Follow up issue to update conceptual docs and structure: https://issues.redhat.com/browse/OBSDOCS-760